Sources/Segment/Configuration.swift

```diff
@@ -30,6 +30,7 @@ public class Configuration {
         var autoAddSegmentDestination: Bool = true
         var apiHost: String = HTTPClient.getDefaultAPIHost()
         var cdnHost: String = HTTPClient.getDefaultCDNHost()
+        var maximumLogFilesOnDisk: Int = 15
```

Comment:
No idea what the right number is here, happy to change the default...

Reply:
We're currently configured to flush at every 3 events (we should probably change this, it feels aggressive and overly chatty) and/or every 10 seconds. 

This means with these defaults we'll have a backlog of 45 events spread over 15 files on disk.